### ISSUE TITLE
rule: collapsed-table-no-border-spacing

### ISSUE THREAD

**Issue by purupurupu** [purupurupu/css-noop-checker#127]
## Rule ID
`collapsed-table-no-border-spacing`

## What does this rule detect?
`border-spacing` only applies when `border-collapse` is `separate` (the default). When `border-collapse: collapse` is set, `border-spacing` is completely ignored.

## Example: should warn
```html
<table style="border-collapse: collapse; border-spacing: 10px;">
  <tr><td>cell</td></tr>
</table>
```

## Example: should NOT warn
```html
<table style="border-collapse: separate; border-spacing: 10px;">
  <tr><td>cell</td></tr>
</table>
```

## Related CSS Properties
`border-collapse`, `border-spacing`

## References
- [CSS 2.2 §17.6.1 - The separated borders model](https://www.w3.org/TR/CSS22/tables.html#separated-borders): border-spacing only applies in separated borders model
- [Firefox inactive-property-helper.js](https://searchfox.org/mozilla-central/source/devtools/server/actors/utils/inactive-property-helper.js) — detects this pattern
